_tools/release_docs/main.go

```diff
@@ -151,6 +158,27 @@ func rewriteCodeBlocks(fileBytes []byte) []byte {
 	return []byte(strings.Join(output, "\n") + "\n")
 }
 
+var (
+	titleRE   = regexp.MustCompile(`# (.*)`)
+	ignoredRE = regexp.MustCompile(`[*_\\]`)
+)
+
+func getTitle(fileBytes []byte) string {
+	match := titleRE.FindSubmatch(fileBytes)
+	if len(match) != 2 {
+		return ""
+	}
+
+	title := match[1]
+	match = linkRE.FindSubmatch(title)
+
+	if len(match) == 3 {
+		title = match[1]
+	}
+
+	return string(ignoredRE.ReplaceAll(title, nil))
```

Comment:
You can also add emphasis and bolding in headers, but the raw markdown would render in the title.